models/TranscribeUtteranceAllOfRecord.ts

```diff
@@ -40,7 +40,7 @@ export class TranscribeUtteranceAllOfRecord {
             "format": "",
 
             
-            "defaultValue": undefined
+            "defaultValue": false
```

Comment:
Wondering if we need AllOf in this filename since we are removing AllOf as a part of this major change?